src/tools/devtools/release-notes/release-notes-2.7.0-src.md

```diff
@@ -53,8 +53,6 @@ in the list of available user tags.
 
 ![Class names]({{site.url}}/tools/devtools/release-notes/images-2.7.0/image6.png "Class names")
 
-## Changelog
-
-More details about changes and fixes are available in our [changelog][].
-
-[changelog]: https://github.com/flutter/devtools/blob/master/CHANGELOG.md
+## Full commit history
+More details about changes and fixes are available from the
+[DevTools git log.](https://github.com/flutter/devtools/commits/master).
```

Comment:
**issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). Write in active voice per [GSG](https://developers.google.com/style/voice). Use site variable.
```suggestion
To learn more about DevTools changes and fixes, check out the
[commit history]({{site.github}}/flutter/devtools/commits/master).
```